src/clients/import/ImportForm.tsx

```diff
@@ -49,7 +51,7 @@ export const ImportForm = () => {
 
   const save = async () => {
     try {
-      await httpClient.doPost("/admin/realms/master/clients", client);
+      await httpClient.doPost(`/admin/realms/${realm}/clients`, client);
```

Comment:
Wouldn't be better to generate these things with some Codegen? I know that keycloak is trying to add Swagger, but even before getting that you can prepare yourself to remove the manual generated SDK.

And also abstract the HTTP/Transport layer out of the UI.

```suggestion
      await sdk.getClients(realm);
```

Reply:
Right there is no Swagger yet, so in the mean time we use http, I don't see a huge benefit of trying to abstract that.